demos/src/scss/_demos.scss

```diff
@@ -48,10 +48,9 @@ body:before {
 }
 
 .o-grid-container {
-	background: lighten(#f6e9d8, 8.5%);
+	background: #EAE8DF;
 }
```

Comment:
These backgrounds have proven super useful when doing visual tests, maybe it'd be best to make a special case for the bleed modifier that removes the background on the sides and the direct children rows?